packages/menlo/data_stream/web/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,319 @@
+---
+description: Mapping of Menlo Web Proxy Fields from API into ECS fields.
+processors:
+  - set:
+      field: ecs.version
+      value: 8.11.0
+  - rename:
+      field: message
+      target_field: event.original
+      ignore_missing: true
+  - set:
+      field: event.kind
+      value: event
+  - json:
+      field: event.original
+      tag: 'json_decoding'
+      target_field: json
+  - date:
+      field: "json.event.event_time"
+      formats:
+        - "ISO8601"
+      target_field: "@timestamp"
+      timezone: "UTC"
+      on_failure:
+        - append:
+            field: error.message
+            value: 'Processor {{{_ingest.on_failure_processor_type}}} with tag fail-{{{_ingest.on_failure_processor_tag}}} in pipeline {{{_ingest.pipeline}}} failed with message: {{{_ingest.on_failure_message}}}'
+  - set:
+      field: "event.module"
+      value: "menlo"
+      ignore_failure: true
+  - convert:
+      field: "json.event.severity"
+      type: long
+      target_field: "event.severity"
+      ignore_missing: true
```

```suggestion
  - convert:
      field: json.event.severity
      type: long
      target_field: event.severity
      ignore_missing: true
```

Remove double-quotes from all field names inside pipeline .